moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -259,131 +265,115 @@ const ProjectActivityLog = () => {
+                                              changeItem.field
+                                            )}
+                                          </span>
+                                          &nbsp;from&nbsp;
+                                          <span className={classes.boldText}>
+                                            {isFieldMapped(
+                                              change.record_type,
+                                              changeItem.field
+                                            )
+                                              ? getMappedValue(
+                                                  change.record_type,
+                                                  changeItem.field,
+                                                  String(changeItem.old)
+                                                )
+                                              : fieldFormat(changeItem.old)}
+                                          </span>
+                                          &nbsp;to&nbsp;
+                                          <span className={classes.boldText}>
+                                            {isFieldMapped(
+                                              change.record_type,
+                                              changeItem.field
+                                            )
+                                              ? getMappedValue(
+                                                  change.record_type,
+                                                  changeItem.field,
+                                                  String(changeItem.new)
+                                                )
+                                              : fieldFormat(changeItem.new)}
+                                          </span>
+                                        </>
+                                      )}
+                                    </Grid>
+                                  );
+                                })}
+                              </Grid>
+                            </Box>
                           </Box>
-                        </Box>
+                        )}
                       </TableCell>
-                      {
-                        // <TableCell width="5%">
-                        //   <Button
-                        //     onClick={() => handleDetailsOpen(change.activity_id)}
-                        //   >
-                        //     Details
-                        //   </Button>
-                        // </TableCell>
-                      }
                     </TableRow>
                   );
                 })}
               </TableBody>
             </Table>
           </TableContainer>
         )}
-        {/*        {!!activityId && (
-          <ProjectActivityLogDialog
-            activity_id={activityId}
-            handleClose={handleDetailsClose}
-          />
-        )}*/}
```

Comment:
there's a lot of great code slashing in this render! looking forward to getting all of the tables up to this new standard so we can continue to pare it down 🚀